devel/install-local-tools.sh

```diff
@@ -214,20 +237,20 @@ function install-go-tools
     }
 
     # Install godoc
-    verbose go get -u -v golang.org/x/tools/cmd/godoc || die "Installing godoc"
+    (cd && GO11MODULE=on verbose go get -u -v golang.org/x/tools/cmd/godoc) || die "Installing godoc"
```

Comment:
INFO Installing on this way, the binary is installed into our GOPATH directory and doesn't modify our go.mod file into our project.

More information here: https://maelvls.dev/go111module-everywhere/ 

FIXME Typo; it should be `GO111MODULE`  instead of `GO11MODULE`